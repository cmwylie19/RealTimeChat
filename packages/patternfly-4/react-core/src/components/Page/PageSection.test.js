"use strict";
exports.__esModule = true;
var React = require("react");
var enzyme_1 = require("enzyme");
var PageSection_1 = require("./PageSection");
jest.mock('./Page');
test('Check page section with no padding example against snapshot', function () {
    var Section = <PageSection_1.PageSection noPadding/>;
    var view = enzyme_1.mount(Section);
    expect(view).toMatchSnapshot();
});
test('Check page main nav section against snapshot', function () {
    var Section = <PageSection_1.PageSection type={PageSection_1.PageSectionTypes.nav}/>;
    var view = enzyme_1.mount(Section);
    expect(view).toMatchSnapshot();
});
test('Check page section with no padding on mobile example against snapshot', function () {
    var Section = <PageSection_1.PageSection noPaddingMobile/>;
    var view = enzyme_1.mount(Section);
    expect(view).toMatchSnapshot();
});
test('Check page section with no fill example against snapshot', function () {
    var Section = <PageSection_1.PageSection isFilled={false}/>;
    var view = enzyme_1.mount(Section);
    expect(view).toMatchSnapshot();
});
test('Check page section with fill example against snapshot', function () {
    var Section = <PageSection_1.PageSection isFilled/>;
    var view = enzyme_1.mount(Section);
    expect(view).toMatchSnapshot();
});
test('Check page section with fill and no padding example against snapshot', function () {
    var Section = <PageSection_1.PageSection isFilled noPadding/>;
    var view = enzyme_1.mount(Section);
    expect(view).toMatchSnapshot();
});
