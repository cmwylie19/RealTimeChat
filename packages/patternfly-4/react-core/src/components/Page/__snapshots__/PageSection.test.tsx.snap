// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check page main nav section against snapshot 1`] = `
<Component
  type="nav"
>
  <section
    className="pf-c-page__main-nav"
  />
</Component>
`;

exports[`Check page section with fill and no padding example against snapshot 1`] = `
<Component
  isFilled={true}
  noPadding={true}
>
  <section
    className="pf-c-page__main-section pf-m-no-padding pf-m-fill"
  />
</Component>
`;

exports[`Check page section with fill example against snapshot 1`] = `
<Component
  isFilled={true}
>
  <section
    className="pf-c-page__main-section pf-m-fill"
  />
</Component>
`;

exports[`Check page section with no fill example against snapshot 1`] = `
<Component
  isFilled={false}
>
  <section
    className="pf-c-page__main-section pf-m-no-fill"
  />
</Component>
`;

exports[`Check page section with no padding example against snapshot 1`] = `
<Component
  noPadding={true}
>
  <section
    className="pf-c-page__main-section pf-m-no-padding"
  />
</Component>
`;

exports[`Check page section with no padding on mobile example against snapshot 1`] = `
<Component
  noPaddingMobile={true}
>
  <section
    className="pf-c-page__main-section pf-m-no-padding-mobile"
  />
</Component>
`;
